SMTPServer/MailInfo.cpp

```diff
@@ -0,0 +1,54 @@
+// This is an independent project of an individual developer. Dear PVS-Studio, please check it.
+
+// PVS-Studio Static Code Analyzer for C, C++, C#, and Java: http://www.viva64.com
+
+#include "MailInfo.h"
+
+void MailInfo::set_mail_from(std::string& mail_from)
```

Comment:
If we get arguments by  reference and will not change it, we can write "const std::string& mail_from".